package.json

```diff
@@ -116,7 +117,8 @@
     "react-native": "0.59.0",
     "react-native-linear-gradient": "^2.5.3",
     "react-test-renderer": "16.8.3",
-    "tmp": "^0.1.0"
+    "tmp": "^0.1.0",
+    "util": "^0.12.0"
```

Comment:
This fixes this issue:

```
Looking for JS files in
   /Users/shaundonnelly/projects/backpack-react-native

Loading dependency graph, done.
::1 - - [07/Jun/2019:14:16:16 +0000] "GET /onchange HTTP/1.1" - - "-" "native/1 CFNetwork/975.0.3 Darwin/18.2.0"
error: bundling failed: Error: Unable to resolve module `util` from `/Users/shaundonnelly/projects/backpack-react-native/packages/react-native-bpk-component-banner-alert/src/customPropTypes.js`: Module `util` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
```